sdk/test-utils/recorder/samples-dev/simpleSample.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Slightly off-topic: I'm curious what the tooling story is with the public recorder? Will we be providing helpers so it's easy for people to run the test proxy in conjunction with their tests (a la dev-tool) or will we just be telling people "hey make sure the test proxy is running on port 5000 before you start your tests"?

Reply:
I'm sure you're considering this already, but I also think we have to do something about the docs story - at the moment all we really have is the migration guide, which is fine if you're migrating a package to the new recorder, but isn't very helpful for onboarding customers :)

Reply:
> I'm curious what the tooling story is with the public recorder? Will we be providing helpers so it's easy for people to run the test proxy in conjunction with their tests (a la dev-tool) or will we just be telling people "hey make sure the test proxy is running on port 5000 before you start your tests"?

dev-tool isn't published, and since the test-proxy is public anyway, it's easier to ask users to run the docker command or the dotnet tool as expected by the recorder.

Reply:
> I'm sure you're considering this already, but I also think we have to do something about the docs story - at the moment all we really have is the migration guide, which is fine if you're migrating a package to the new recorder, but isn't very helpful for onboarding customers :)